FinniversKit/Sources/DNA/Color/ColorProvider.swift

```diff
@@ -69,6 +71,10 @@ public struct DefaultColorProvider: ColorProvider {
         .dynamicColor(defaultColor: .aqua50, darkModeColor: .darkIce)
     }
 
+    public var bgInfoHeader: UIColor {
+        .dynamicColor(defaultColor: .accentToothpaste, darkModeColor: .accentPrimaryBlue)
```

Comment:
To make things extra complicated, when you define the semantic color here you should probably rather not use other semantic colors, so here I would rather go for the values those colors have (`.aqua200` and `.blue600` I think)